requirements.txt

```diff
@@ -48,3 +48,4 @@ cognitiveatlas
 django-datatables-view
 git+git://github.com/vsoch/nidmviewer.git@0.1
 https://github.com/benkonrath/django-guardian/archive/django-polymorphic.zip
+django-oauth-toolkit==0.9.0
```

Comment:
could you add this to circleci as well? (we have this redundancy to speed up builds)